### PR TITLE
[release/dev16.10] update dev16.10 insertion target

### PR DESCRIPTION
**Pull request by brettfo**

### CODE REVIEW

**Comment by cartermp** [comment]
Let's hold off until this last merge to 16.10 today :)

**Review by vzarytovskii — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Comment by cartermp** [comment]
All done

**Pull request merged**
